### PR TITLE
feat: add i18n support

### PR DESCRIPTION
**Pull request by olizilla**
Add localisation support via i18next and transifex. You can now run

```console
npm run storybook
```
and see new stories for ExplorePage and StartExploringPage complete with translations. Only the default `en` phrases are available currently, but there is a project on Transifex for it: https://www.transifex.com/ipfs/ipld-explorer/dashboard/

Changes:

- Add transifex config in .tx
- Replace text with calls to t()
- Add storybooks for page components
- Add i18n decorator to easily add i18next provider to a story
- Add bundle decorator to allow stubbing out of redux-bundler selectors
- Add action creators to explore bundle for responding to path changes and following links

screenshot 2018-09-06 17 43 09

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

File: src/components/object-info/ObjectInfo.js [hunk 1]
```diff
@@ -71,7 +72,7 @@ const ObjectInfo = ({ className, type, cid, localPath, size, data, links, format
         ) : null}
         {format === 'unixfs' && data.type && ['directory', 'file'].some(x => x === data.type) ? (
           <a className='link avenir ml2 pa2 fw5 f6 blue' href={`https://ipfs.io/ipfs/${cid}`} target='_external'>
-            View on IPFS Gateway
+            {t('')}
```
Comment (lidel) [inline src/components/object-info/ObjectInfo.js]: what is happening here :thinking:  ?

File: src/components/about/AboutIpld.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export const AboutIpld = () => {
+  return (
+    <Box className='tl dib pa4 avenir measure-wide-l lh-copy dark-gray ba-l b--black-10'>
+      <div className='tc'>
+        <a className='link' href='https://ipld.io'>
+          <img src={ipldLogoSrc} alt='IPLD' style={{ height: 60 }} />
+        </a>
+      </div>
+      <Trans i18nKey='AboutIpld.paragraph1'>
```
Comment (lidel) [inline src/components/about/AboutIpld.js]: This is a really cool approach! :heart:

**Comment by hacdias** [comment]
Working on PT translation.

**Comment by hacdias** [comment]
@olizilla just added PT translation.

image

Using a fixed width is bad for translations. Unfortunately, we have more examples of these across the WebUI. We should take a look at this better. For now, you could add just some pixels more or maybe use flexbox.

File: src/components/object-info/ObjectInfo.js (hunk 1 above)
Comment (olizilla) [reply src/components/object-info/ObjectInfo.js]: MISSING KEY IS MISSING! @lidel @hacdias should I add the `en` value to the translation files or can this be automagically added later by a `tx pull`?

**Comment by olizilla** [comment]
@hacdias yep, when I built that form I was trying to always use the "oldest" mechanism that would work... `display: table` has awesome browser support, but you are correct, the fixed width is problematic here. I'll switch it to flexbox.

File: src/components/object-info/ObjectInfo.js (hunk 1 above)
Comment (lidel) [reply src/components/object-info/ObjectInfo.js]: It will be added automatically next time you do `tx pull -a`

**Pull request merged**